src/ucp/wireup/wireup_cm.c

```diff
@@ -1199,7 +1213,7 @@ ucp_ep_server_init_priv_data(ucp_ep_h ep,  const char *dev_name,
     ucp_tl_bitmap_validate(&tl_bitmap, &ctx_tl_bitmap);
 
     dev_index = ucp_cm_tl_bitmap_get_dev_idx(worker->context, &tl_bitmap);
-    status    = ucp_cm_ep_priv_data_pack(ep, &tl_bitmap, dev_index,
+    status    = ucp_cm_ep_priv_data_pack(ep, &tl_bitmap, dev_index, 0,
```

Comment:
before this PR, can_fallback was equivalent to ucp_cm_client_get_next_cm_idx(ep)